agent/api/task/task_linux.go

```diff
@@ -355,3 +357,37 @@ func (task *Task) BuildCNIConfig(includeIPAMConfig bool, cniConfig *ecscni.Confi
 
 	return cniConfig, nil
 }
+
+// BuildCNIConfigBridgeMode builds a list of CNI network configurations for a task in docker bridge mode.
+// Currently the only plugin in available is for Service Connect
+func (task *Task) BuildCNIConfigBridgeMode(cniConfig *ecscni.Config, containerName string) (*ecscni.Config, error) {
```

Comment:
I separated bridge-mode CNI config creation from the AWSVPC because they are very different - bridge-mode only needs the ServiceConnect CNI plugin, while AWSVPC need a bunch more, and other configurations. 

If we are introducing more network modes that will use CNI plugins, then we could combine it with the existing AWSVPC or bridge CNI config build if there's a lot of similarity, or we could create a new one for its own.